Added docs for numbers in jsescOption in babel-generator (#2172)
babel/website: website/versioned_docs/version-7.0.0/generator.md

concise                | boolean  | `false`         | Set to `true` to reduce whitespace (but not as much as `opts.compact`)
filename               | string   |                 | Used in warning messages
jsonCompatibleStrings  | boolean  | `false`         | Set to true to run `jsesc` with "json": true to print "\u00A9" vs. "©";

> jsescOption            | object   |                 | Use `jsesc` to process literals. `jsesc` is applied to numbers only if `jsescOption.numbers` is present. You can customize `jsesc` by [passing options](https://github.com/mathiasbynens/jsesc#api) to it.

Reviewer: (This is for versions _before_ 7.9.0)

```suggestion
jsescOption            | object   |                 | Use `jsesc` to process string literals. You can customize `jsesc` by [passing options](https://github.com/mathiasbynens/jsesc#api) to it.
```
